sonar/modules/documents/api.py

```diff
@@ -49,6 +51,50 @@ class DocumentRecord(SonarRecord):
     fetcher = document_pid_fetcher
     provider = DocumentProvider
     schema = 'document'
+    affiliations = []
```

Comment:
With my little experience, affiliations should be a dedicated ressource in next development :-) (with key: main institution, dpt-name, start-date/end-date, parent_affiliation ($ref on same ressource)), ...

Reply:
I'm sure you're right, but it's a PO consideration and I will discuss with them later.